contrib/kubernetes/datahub/Chart.yaml

```diff
@@ -4,24 +4,24 @@ description: A Helm chart for LinkedIn DataHub
 type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 0.0.1
+version: 0.1.0
```

Comment:
I have limited knowledge in helm, but any reason why we're not aligning this version with individual dependency versions (i.e. `0.2.0`)?

Reply:
Each chart (4 charts for gms services and 1 composing chart called datahub) is an independent entity in itself and can maintain its own versions. The composing chart, "datahub" can subscribe to any version of sub charts and thus can maintain its own versioning.
For example, I did some minor change only to chart for "datahub-gms" and thus upgrade its version to 0.2.1. Other sub charts version will still be 0.2.0. The composing chart can be upgrade to 0.1.1.
Similarly, I can add another sub-chart for any new service in future. It's first version will be 0.1.0, though rest of the sub-charts will be much higher.
In short, it is like Maven dependency management, transitive dependency version has no correlation to composing dependency.

Thanks.

Reply:
Got it. As long as this is following the helm chart convention which sounds like it is.